src/shared/components/syntax-highlighter/index.js

```diff
@@ -6,12 +6,12 @@ import classNames from 'classnames'
 import 'prismjs/themes/prism-okaidia.css'
 import styles from './index.module.css'
 
-const SyntaxHighlighter = ({ codeStr, language }) => {
-  const htmlCode = Prism.highlight(codeStr, Prism.languages.js, 'js')
+const SyntaxHighlighter = ({ codeStr, language, className }) => {
+  const htmlCode = Prism.highlight(codeStr, Prism.languages.js, language)
   const preClasses = classNames(`language-${language}`, styles.preCustom)
 
   return (
-    <div className={ styles.gatsbyHighlight } >
+    <div className={ styles.gatsbyHighlight + ' ' + className } >
```

Comment:
Use the `classnames` module here as well.

Reply:
You are right, it's not necessary to use classnames here.

Reply:
I think that `classNames` is a cleaner approach and we use it across the project, even for this type of classNames with no conditionals